testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LoginTest.java

```diff
@@ -363,19 +359,19 @@ public void loginDifferentUserAfterDisabledUserThrownOut() {
             appPage.assertCurrent();
             appPage.openAccount();
 
-            profilePage.assertCurrent();
-
             setUserEnabled(userId, false);
 
             // force refresh token which results in redirecting to login page
-            profilePage.updateUsername("notPermitted");
+            AccountHelper.updateUsername(adminClient.realm("test"), "test-user@localhost", "notPermitted");
```

Comment:
Not sure this is actually doing anything relevant. Previously it was updating the username only for forcing the refresh token. This caused a redirect to login page. Now you are just setting a new username and manually opening the login page, with no refresh token update.

Reply:
Yes, but I thought the updateUsername on the API also refreshed the token. If it is not like that I'll remove it